operate/importer-8_5/src/main/java/io/camunda/operate/zeebeimport/processors/fni/FNITransformer.java

```diff
@@ -33,10 +34,18 @@ public class FNITransformer {
   private static final Set<String> FINISH_STATES =
       Set.of(ELEMENT_COMPLETED.name(), ELEMENT_TERMINATED.name());
   private static final Set<String> START_STATES = Set.of(ELEMENT_ACTIVATING.name());
-  private final Function<FNITreePathCacheCompositeKey, String> treePathResolver;
 
-  public FNITransformer(final Function<FNITreePathCacheCompositeKey, String> treePathResolver) {
-    this.treePathResolver = treePathResolver;
+  private static final Set<BpmnElementType> CONTAINER_TYPES =
+      Set.of(
+          BpmnElementType.CALL_ACTIVITY,
```

Comment:
If I am not mistaken, then a call activity doesn't have a "child" flow node instance (i.e., element instance). So, in that sense, it is not a container type. It might only be relevant if that cache is used to store the tree path containing the call hierarchy.

Reply:
Yeah here I was not sure, and wanted to first add all I know. Happy to remove it.

Reply:
Should I remove it? I think it will be added to the cache at least before 🤔 The only point where call activity are mentioned is here https://github.com/camunda/camunda/blob/main/operate/importer-8_6/src/main/java/io/camunda/operate/zeebeimport/processors/ListViewZeebeRecordProcessor.java#L722 but maybe I missed something

Reply:
Yes, let's remove 😄 Besides that, the PR looks good.

Reply:
Done 👍🏼